data/json/monsters/feral_humans.json

```diff
@@ -1019,5 +1019,119 @@
       "PATH_AVOID_DANGER_1",
       "DROPS_AMMO"
     ]
+  },
+  {
+    "id": "mon_feral_big_pappy",
+    "type": "MONSTER",
+    "name": { "str": "Big Pappy", "str_pl": "Big Pappys" },
```

Comment:
```suggestion
    "name": { "str": "Big Pappy" },
```
`s` is added automatically.

Reply:
Yep - it failed it again haha

Reply:
Possibly because it ends in a `y`?

Reply:
Let me know if you want me to re-add it.

Reply:
Done!